### ISSUE TITLE
Improve naming and be consistent about it

### ISSUE THREAD

**Issue by sbs2001** [nexB/vulnerablecode#447]
Currently we use different terms to describe the same thing 
eg safe_package, patched_package, resolved_package . 

This should be avoided. We should stick to something once for all.

The https://github.com/nexB/vulnerablecode/blob/fd1572438831ff41cfc856da5d3194b5f2ef825b/vulnerabilities/helpers.py#L124 will also need a good name. 
See the comments at https://github.com/nexB/vulnerablecode/pull/436 for more details

**Comment by MUzairS15** [comment]
Means wherever we encounter these safe_package, patched_package, resolved_package, we need to replace them. But by which common name, also will it not conflict other parts.? I am new pls guide me?

**Comment by pombredanne** [comment]
@MUzairS15 re:

> Means wherever we encounter these safe_package, patched_package, resolved_package, we need to replace them. But by which common name, also will it not conflict other parts.?

yes.

The first step is to find the good names alright!



We would need to survey the words used to depict affected/patched in other tools:

- kaybee/vulas
- dependencycheck and dependency track (and OWASP)
- in general all the data source we use
    ...
 with these listed here we can make a good pick together!

**Comment by MUzairS15** [comment]
Ok.
So can you tell from where i can start with?

**Comment by MUzairS15** [comment]
Scan the whole code make a list and choose one from there and apply changes right?

**Comment by pombredanne** [comment]
> Scan the whole code make a list and choose one from there...

yes, that would be a start: a survey of the terms used in the external data sources we use in the importers here and some research outside. And posting the results of this research here before we do anything else.

**Comment by DennisClark** [comment]
We have standardized on field names after extensive discussion.  Closing.
